chef_master/source/aws_opsworks_chef_automate.rst

```diff
@@ -68,3 +68,96 @@ Enabling the compliance profile storage service in AWS OpsWorks for Chef Automat
 #. `Configure the audit cookbook <https://github.com/chef-cookbooks/audit#reporting-to-chef-visibility-via-chef-server>`_ to scan your nodes.
 
 .. note:: These instructions only detail what has to be added to the existing configuration as found in AWS OpsWorks for Chef Automate. For general instructions, see :doc:`integrate_compliance_chef_automate` (collector ``chef-server-visibility``) and :doc:`install_chef_automate`.
+
+Adding Push Jobs Server based build nodes to AWS OpsWorks for Chef Automate
+============================================================================
+
+Adding Push Jobs Server based build nodes to AWS OpsWorks for Chef Automate requires changes to the Chef Automate instance as well as the node that will run as a builder.
+
+#. Install Push Jobs Server on the Chef Automate instance by downloading the Push Jobs Server from https://downloads.chef.io/push-jobs-server and running:
```

Comment:
@srenatus: From @sersut's notes:

> Normally these steps are automated via `automate-ctl install-build-node` command but the version of Opsworks Automate is 0.6.7 and it does not include a fix that went in after this version which makes this command work correctly in the proxied Chef Server environment.
> Because we do not have this fix we need to do the below steps in order to make push jobs based builders work. Otherwise you will have a failure as such:

```
$ sudo automate-ctl install-build-node --fqdn ec2-52-23-235-94.compute-1.amazonaws.com --username centos  --ssh-identity-file /home/ec2-user/id_rsa --installer /home/ec2-user/chefdk-1.1.16-1.el6.x86_64.rpm
...
...
[2016-12-20T20:46:26.608590 #31905] FATAL -- : Error bootstrapping ec2-52-23-235-94.compute-1.amazonaws.com: /opt/delivery/embedded/bin/knife search node '(name:ec2-52-23-235-94.compute-1.amazonaws.com OR fqdn:ec2-52-23-235-94.compute-1.amazonaws.com OR ipaddress:ec2-52-23-235-94.compute-1.amazonaws.com) AND role:delivery_builders' -a fqdn -u delivery -k /etc/delivery/delivery.pem --server-url https://localhost:8443/organizations/default returned 100
...
...
ERROR: SSL Validation failure connecting to host: localhost - hostname "localhost" does not match the server certificate
ERROR: SSL Error connecting to https://localhost:8443/organizations/default/search/node?q=(name:ec2-52-23-235-94.compute-1.amazonaws.com%2520OR%2520fqdn:ec2-52-23-235-94.compute-1.amazonaws.com%2520OR%2520ipaddress:ec2-52-23-235-94.compute-1.amazonaws.com)%2520AND%2520role:delivery_builders&sort=X_CHEF_id_CHEF_X%20asc&start=0, retry 1/5
...
...
```